### PR TITLE
Fix windows paths in cloud dir

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 55ac1449ea8641cb7107715a7b94695b6f696bcd:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-b3cf2d8f391fe575dca1d9173994706f0fd45f4d-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-b3cf2d8f391fe575dca1d9173994706f0fd45f4d-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-b3cf2d8f391fe575dca1d9173994706f0fd45f4d-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-b3cf2d8f391fe575dca1d9173994706f0fd45f4d-x86.apk)

**Review by nirvn — APPROVED** [review]

File: src/core/utils/qfieldcloudutils.cpp [hunk 1]
```diff
@@ -59,8 +61,10 @@ bool QFieldCloudUtils::isCloudAction( const QgsMapLayer *layer )
 const QString QFieldCloudUtils::getProjectId( const QString &fileName )
 {
   QFileInfo fi( fileName );
-  if ( fileName.startsWith( localCloudDirectory() ) )
-    return fi.dir().dirName();
+  QDir baseDir = fi.dir();
+  QString basePath = QDir::cleanPath( baseDir.path() );
```
Comment (m-kuhn) [inline src/core/utils/qfieldcloudutils.cpp:65]:
```suggestion
  QString basePath = QDir::cleanPath( baseDir.path() );
  qWarning() << "Base path " << basePath;
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d103943dcf0adfc5b98598ceb0222da37a099253:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-3c16a52d5263c48154ca5ce6f630dc4bb2694dfe-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-3c16a52d5263c48154ca5ce6f630dc4bb2694dfe-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-3c16a52d5263c48154ca5ce6f630dc4bb2694dfe-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-3c16a52d5263c48154ca5ce6f630dc4bb2694dfe-x86.apk)

**Comment by m-kuhn** [comment]
Base path still misses drive letter

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for ca1f9c3ff3d5f5a56f5e344ab7bf7712ca7ce728:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-d4bc99dcccf548cd06ccb48f6ad2a6f6887aa10f-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-d4bc99dcccf548cd06ccb48f6ad2a6f6887aa10f-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-d4bc99dcccf548cd06ccb48f6ad2a6f6887aa10f-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-d4bc99dcccf548cd06ccb48f6ad2a6f6887aa10f-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for c25858407bca6c907f97b12aea9b69813878e43b:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-415bbc72c1d899eea3647eaa294d47781bb534e6-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-415bbc72c1d899eea3647eaa294d47781bb534e6-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-415bbc72c1d899eea3647eaa294d47781bb534e6-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-415bbc72c1d899eea3647eaa294d47781bb534e6-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 817af6482f1c8ad97a1c22af2dca39b2d13c8c70:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-2755a26624561f66843cb2479940e8c387bac692-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-2755a26624561f66843cb2479940e8c387bac692-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-2755a26624561f66843cb2479940e8c387bac692-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-2755a26624561f66843cb2479940e8c387bac692-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a6b88cdbf7c8a790f39933d352d5a84dfb01c023:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-ed517c17e70724690a6c87459f339c524c24a5a3-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-ed517c17e70724690a6c87459f339c524c24a5a3-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-ed517c17e70724690a6c87459f339c524c24a5a3-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2129-ed517c17e70724690a6c87459f339c524c24a5a3-x86.apk)

**Pull request merged**